arlo-client/src/components/Audit/StatusBox/getJurisdictions.ts

```diff
@@ -0,0 +1,47 @@
+import { toast } from 'react-toastify'
+import { api, checkAndToast } from '../../utilities'
+import { IErrorResponse } from '../../../types'
+
+export enum FileProcessingStatus {
+  Blank = 'NULL', // only returned from getJurisdictionFileStatus, represents the null state from the server
+  ReadyToProcess = 'READY_TO_PROCESS', // only received from the server, never returned from getJurisdictionFileStatus; equivalent to 'Processing'
+  Processing = 'PROCESSING',
+  Processed = 'PROCESSED',
+  Errored = 'ERRORED',
+}
+
+export interface IJurisdictionsResponse {
+  jurisdictions: {
+    id: string
+    name: string
+    ballotManifest: {
+      file: string | null
+      processing: FileProcessingStatus | null
+      numBallots: number | null
+      numBatches: number | null
+    }
+    currentRoundStatus: number | null
+  }[]
+}
+
+const getJurisdictions = async (
+  electionId: string
+): Promise<undefined | IJurisdictionsResponse> => {
+  try {
+    const jurisdictionsOrError:
+      | IJurisdictionsResponse
+      | IErrorResponse = await api(`/election/${electionId}/jurisdiction`)
+    // checkAndToast left here for consistency and reference but not tested since it's vestigial
```

Comment:
Is it really vestigial? If so, should it be removed?

Reply:
@benadida We set up the `checkAndToast` error handling pattern on the frontend for managing errors sent from the server, but it doesn't match how we're doing errors on the server anymore. So we need to sit down and unify how that's going to be handled, but not just yet. I anticipate that it will still use this pattern on the api call itself, just change the internal functionality of the `checkAndToast` function, though, so that's why it's still here. I add it in so that if the function just needs an internal change, it will still work out of the gate, or also if we need to change the implementation as well, it's an easy search and replace to change them across the whole codebase. So it's vestigial currently, but might be resuscitated, if that makes sense.

Reply:
Gotcha. I would still recommend removing it because otherwise there's a chance it stays and we never remove it, but your call.

Reply:
Just created an issue so I wouldn't forget, haha. https://github.com/votingworks/arlo/issues/454